### DISCUSSION TITLE
intercepting Android remote control keys, etc.

### DISCUSSION THREAD

**Discussion by firemen01** [DelphiWorlds/Kastri#240] [Ideas]
Delphi does not support keys, for example the remote control (OK-23, back-137, etc.). It would be nice to implement this.
the example in the attachment.

[FMX.KeyMapping.zip](https://github.com/user-attachments/files/16522714/FMX.KeyMapping.zip)

**Comment by DelphiWorlds** [comment]
I have something in the works for this
